src/App.js

```diff
@@ -1,24 +1,33 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
```

Comment:
- [ ] When adding a new car, the image is inconsistent with the others:

![image](https://github.com/ivangonzalez224/Tesla-car-booking-front-end/assets/57421823/6b1083ed-1309-4bc1-b47a-a4283ed10762)

Kindly, add some boundaries to your CSS styling, so any user added image is the same size as the others. 👍